documentation/modules/creating-migration-quick-plan-new-source.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-new-source_{context}"]
+= Creating a migration quick plan with a new source provider
+
+You can create a new source proider as part of creating a migration _quick plan_.
+
+.Prerequisite
+
+* There is at least one user-defined destination provider.
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization* and then click *Create Provider*.
+. Create a source provider following the instructions in "Adding Source providers" TBD
++
```

Comment:
@anarnold97 Here, too, please.

Reply:
[id="adding-source-providers"]
==== Adding source providers

in master.adoc

Shortly after the header and loist of providers we get into the contexts (in master.adoc) and ifdef statements in adding-source-provider.adoc, so I think it would be best to just refer to to the header in master.adoc,